mavlink.c

```diff
@@ -127,6 +127,14 @@ void* __MAVLINK_THREAD__(void* arg) {
           case MAVLINK_MSG_ID_HEARTBEAT:
             // handle_heartbeat(&message);
             break;
+		
+          case MAVLINK_MSG_ID_RAW_IMU:
```

Comment:
MAVLINK_MSG_ID_RAW_IMU seems to be for IMU data:
https://docs.ros.org/en/melodic/api/rosflight/html/mavlink__msg__raw__imu_8h.html#a597d5ddd09d163766bb8afe327d12085
isn't there a message for temperature in mavlink ?